### ISSUE TITLE
Error when calling belongsToMany-relationship

### ISSUE THREAD

**Issue by JoeyHoutenbos** [moloquent/moloquent#52]
I am getting this error `Call to undefined method Moloquent\Query\Builder::getBelongsToManyCaller()` when calling ```$menu->organisations()```.

The relationship in the menu-model is defined as:
```php
	public function organisations()
	{
		return $this->belongsToMany(Organisation::class);
	}
```

I am using the "dev-master"-branch.
The method is being called from `Moloquent\Eloquent\HybridRelations` line `234`. The problem seems related to Laravel 5.4 compatibility as discussed here: https://github.com/moloquent/moloquent/pull/49

Work-around for this problem is to define the relationship as:
```php
	public function organisations()
	{
		return $this->belongsToMany(Organisation::class, null, null, null, 'organisations');
	}
```
But when using `attach`-method, this error occurs `Undefined property: Moloquent\Relations\BelongsToMany::$otherKey`.

Also take a look at this: https://github.com/jenssegers/laravel-mongodb/issues/1108

**Comment by SenorWesley** [comment]
This was solved for me, see pull request [moloquent#53](https://github.com/moloquent/moloquent/pull/53)

**Comment by JoeyHoutenbos** [comment]
This PR doesn't solve the problem. See my comment on the PR: https://github.com/moloquent/moloquent/pull/53#issuecomment-290384770
